web/ts/chat/NewChatMessage.ts

```diff
@@ -21,10 +20,25 @@ export class NewChatMessage {
 
     clear(): void {
         this.message = "";
-        this.replyTo = 0;
+        this.reply = undefined;
         this.addressedTo = [];
     }
 
+    setReply(m: ChatMessage) {
+        this.reply = {
+            message: m,
+            id: m.ID,
+        };
+    }
+
+    cancelReply() {
```

Comment:
I think the `cancelReply()` logic might be broken.
There seem to be some ways to get undefined state.

This can be triggered in two ways.
1. If escape or the cross symbol is pressed.
2. After a reply is sent.

If another message is sent now in the normal chat, the error occurs.
Error: `Uncaught TypeError: e.reply is undefined`